### PR TITLE
Fixed loading saved keras model (h5 files) when using advanced activation functions

### PR DESCRIPTION
**Pull request by PiyushDatta**
@k-w-w and @jvishnuvardhan  , reference to #38994 

Please review changes. Tests: activations_test.py and advanced_activations.py passed.

We need to bring in the classes from advanced_activations if there are no custom objects specified. When no custom objects are specified, our module_objects/globals() in activations.deserialize() won't contain any advanced_activations.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -439,9 +440,18 @@ def deserialize(name, custom_objects=None):
       ValueError: `Unknown activation function` if the input string does not
       denote any defined Tensorflow activation function.
   """
+  globs = globals()
+
+  # only replace missing activations, when there are no custom objects
+  if custom_objects is None:
```
Comment (k-w-w) [inline tensorflow/python/keras/activations.py:446]: This if statement can be removed -- [the custom object dict is always checked before the module objects](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/utils/generic_utils.py#L200), so it's safe to add the missing activations here.

**Review by k-w-w — CHANGES_REQUESTED** [review]

**Comment by k-w-w** [comment]
Thanks for adding this!

**Review by PiyushDatta — COMMENTED** [review]
removed if statement since the custom object dict is always checked before the module objects

**Review by k-w-w — DISMISSED** [review]
Thanks!

**Comment by PiyushDatta** [comment]
Sorry, can you review the new changes 5224e939e0a21f88f1aeee11d9477e67d8b9b07f, had to fix pylint issues.

**Review by k-w-w — DISMISSED** [review]

**Review by gbaned — APPROVED** [review]

**Review by gbaned — DISMISSED** [review]

**Comment by k-w-w** [comment]
This is failing the internal builds with the error `ImportError: cannot import name 'advanced_activations'`
Please update the [BUILD](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/BUILD#L113-L123) so that the activations build rule includes `"//tensorflow/python/keras/layers:advanced_activations"` as a dependency.

**Comment by PiyushDatta** [comment]
@k-w-w Done
@gbaned sorry, can you remove all reviewers except @k-w-w , it was asked automatically

**Review by k-w-w — APPROVED** [review]
This looks good, just need to wait for the presubmits to run.

File: tensorflow/python/keras/BUILD [hunk 2]
```diff
@@ -119,6 +119,7 @@ py_library(
     deps = [
         ":backend",
         "//tensorflow/python/keras/utils:engine_utils",
+        "//tensorflow/python/keras/layers:advanced_activations",
```
Comment (k-w-w) [inline tensorflow/python/keras/BUILD:122]: Ah, there's a small issue here. Can you swap lines 121 and 122? (deps should appear in alphabetical order)

**Review by k-w-w — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/BUILD (hunk 2 above)
Comment (PiyushDatta) [reply tensorflow/python/keras/BUILD:121]: ah good to know, sorry about that. Fixed! :)

**Review by k-w-w — APPROVED** [review]

**Pull request merged**